tst/com/amazon/ion/benchmark/DataGeneratorTest.java

```diff
@@ -211,6 +216,51 @@ public void testViolationOfIonBlob() throws Exception {
         DataGeneratorTest.violationDetect(INPUT_ION_BLOB_FILE_PATH);
     }
 
+    /**
+     * Test if there's violation when generating floats from Ion Schema file.
+     * @throws Exception if error occurs during the violation detecting process.
+     */
+    @Test
+    public void testViolationOfFloat() throws Exception {
+        DataGeneratorTest.violationDetect(INPUT_ION_FLOAT_FILE_PATH);
+    }
+
+    /**
+     * Test if there's violation when generating floats from Ion Schema file.
```

Comment:
floats -> symbols

This same copy/paste error occurs on other JavaDoc comments on the test methods in this file.

Reply:
> floats -> symbols
> 
> This same copy/paste error occurs on other JavaDoc comments on the test methods in this file.

Thanks, the updates will be included in the next commit.